mingw-w64-meanwhile/PKGBUILD

```
@@ -7,38 +7,47 @@
 _realname=meanwhile
 pkgbase=mingw-w64-${_realname}
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-pkgver=1.0.2
-pkgrel=5
+pkgver=1.1.1
+pkgrel=1
 pkgdesc="Meanwhile libraries (mingw64)"
```

```suggestion
pkgdesc="Meanwhile libraries (mingw-w64)"
```
please add this change while at it